samtools.wdl

```diff
@@ -1,15 +1,16 @@
 task Index {
     String? preCommand
-    String bamFilePath
+    File bamFilePath
+    String bamIndexPath
```

Comment:
Is it possible to make this an optional string with default `bamFilePath + ".bai"`? Saves having to specify it.

Reply:
Could be optional without a default. Samtools index default output path is already exactly that.
I will change this.

Reply:
:+1: